dbbackup/tests/utils.py

```diff
@@ -10,9 +15,22 @@
 
 GPG_PRIVATE_PATH = os.path.join(settings.BASE_DIR, 'tests/gpg/secring.gpg')
 GPG_PUBLIC_PATH = os.path.join(settings.BASE_DIR, 'tests/gpg/pubring.gpg')
+GPG_FINGERPRINT = '7438 8D4E 02AF C011 4E2F  1E79 F7D1 BBF0 1F63 FDE9'
 DEV_NULL = open(os.devnull, 'w')
```

Comment:
In case you need to verify that data is written correctly and can be restored, you might need a persistent storage? Like `tempfile.mkdtemp`.